skeleton/nix/pkgs.nix

```diff
@@ -33,10 +33,10 @@ let
   pkgSet = haskell.mkStackPkgSet {
     inherit stack-pkgs;
     modules = [
-      # Add source filtering to local packages
+      # Add filtered sources to local packages
       {
         packages.iohk-skeleton.src = src;
-        # packages.another-package = src + /another-package;
+        # packages.another-package.src = src + /another-package;
```

Comment:
Just for future reference I guess this is what people needs to do if the project has multiple directories:

https://github.com/input-output-hk/cardano-ledger-specs/pull/1009/commits/25d7ae13599a45da197783f38c132266c1c94d5f#diff-942fcbe81271545077c2ade39eea9a6eR31

I wonder whether `iohk-nix` should provide such `filterSubDir` function.